libs/sdk-ui-tests/stories/visual-regression/filters/RankingFilter/RankingFilter.tsx

```diff
@@ -38,7 +38,7 @@ const buttonScenarios = {
     opened: { clickSelector: ".s-rf-dropdown-button", postInteractionWait: 200 },
 };
 
-const rankingFilter = newRankingFilter(ExperimentalLdm.Amount_1.Sum, "TOP", 10);
+const rankingFilter = newRankingFilter(ExperimentalLdm.Amount_1.Sum, "TOP", 42);
```

Comment:
I think you need to rebuild scenarios because of this.

Reply:
Good catch, I did not meant to commit this.

Reply:
Reverted.